src/SonarQube.TeamBuild.Integration.Legacy/LegacyTeamBuildFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿/*
+ * SonarQube Scanner for MSBuild
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarQube.TeamBuild.Integration.Legacy
+{
+    public class LegacyTeamBuildFactory : ILegacyTeamBuildFactory
+    {
+        public ILegacyBuildSummaryLogger BuildLegacyBuildSummaryLogger(string tfsUri, string buildUri)
```

Comment:
[cosmetic] You are mixing new syntax (later on) and old syntax (here). I'd rather have something uniform. Full method (for one liners) vs arrowed expression.